browser/js/common/directives/navbar/navbar.html

```diff
@@ -3,19 +3,21 @@
         <div class="navbar-header">
             <fullstack-logo></fullstack-logo>
         </div>
+
         <ul class="nav navbar-nav">
-            <li ng-repeat="item in items" ng-show="!item.auth || isLoggedIn()">
+            <li ng-repeat="item in items" ng-show="!item.auth || isLoggedIn() && user.firstName !== 'Unregistered'">
                 <a ui-sref-active="active" ui-sref="{{ item.state }}">{{ item.label }}</a>
             </li>
-						<li>
-						</li>
         </ul>
-    <!--     <button ng-show="!user" ui-sref="signup" class="btn login-button">Sign Up</button> -->
-        <button ng-show="!user" ui-sref="login" class="btn login-button">Login</button>
-        <div ng-show="user" class="welcome">
-            <span>Welcome, {{ user.email }}!</span>
+
+        <button ng-show="!user || user.firstName === 'Unregistered'" ui-sref="login" class="btn login-button">Login</button>
+
+        <div ng-show="user && user.firstName !== 'Unregistered'" class="welcome">
```

Comment:
at this moment, we move forward with more feature than concerning signing them up. So we did not plan to change the sign up. Can be fix later.